packages/admin/src/Charcoal/Admin/Mustache/AssetsHelpers.php

```diff
@@ -80,13 +81,17 @@ protected function reset()
      * @param  LambdaHelper|null $helper For rendering strings in the current context.
      * @return string
      */
-    public function __invoke($text = null, LambdaHelper $helper = null)
+    public function __invoke($text = null, ?LambdaHelper $helper = null)
     {
         if ($helper) {
             $text = $helper->render($text);
         }
-        $return = $this->{$this->action}($this->collection, $text);
-        $text   = $return;
+
+        if (!empty($this->action)) {
+            $return = $this->{$this->action}($this->collection, $text);
+        }
+
+        $text = ($return ?? '');
```

Comment:
The `empty()` call is unnecessary because the `$action` property is explicitly declared on the class.


```suggestion

        if ($this->action) {
            $text = ($this->{$this->action}($this->collection, $text) ?? '');
        }
```